Tensile/KernelWriter.py

```diff
@@ -1879,6 +1849,14 @@ def initKernel(self, kernel, tensorParametersA, tensorParametersB ):
     vwa = kernel["GlobalLoadVectorWidthA"]
     vwb = kernel["GlobalLoadVectorWidthB"]
 
+    # Wider LocalRead
+    if kernel["MatrixInstruction"]:
```

Comment:
For consistency: EnableMatrixInstruction

Reply:
done